enterprise/backend/test/metabase_enterprise/sandbox/test_util.clj

```diff
@@ -116,6 +116,12 @@
   [gtaps-and-attributes-map & body]
   `(do-with-gtaps-for-user (fn [] ~gtaps-and-attributes-map) :rasta (fn [~'&group] ~@body)))
 
+(defmacro with-gtaps-for-user
```

Comment:
There is already a 
```clojure
(defmacro with-gtaps-for-user
  "Like `with-gtaps`, but for an arbitrary User. `test-user-name-or-user-id` can be a predefined test user e.g. `:rasta`
  or an arbitrary User ID."
  {:style/indent 2}
  [test-user-name-or-user-id gtaps-and-attributes-map & body]
  `(do-with-gtaps-for-user (fn [] ~gtaps-and-attributes-map) ~test-user-name-or-user-id (fn [~'&group] ~@body)))
```
on line 88 of this file.

Reply:
removed the new one.